src/arch/arm/machine/gic_v3.c

```diff
@@ -277,7 +277,7 @@ BOOT_CODE static void cpu_iface_init(void)
 
     /* EOI drops priority and deactivates the interrupt: ICC_CTLR_EL1 */
     SYSTEM_READ_WORD(ICC_CTLR_EL1, icc_ctlr);
-    icc_ctlr &= ~BIT(GICC_CTLR_EL1_EOImode_drop);
```

Comment:
it looks like the EOImodeNS bit in bit 9. please double check.

Reply:
Based on this: https://developer.arm.com/documentation/ddi0595/2021-03/AArch64-Registers/ICC-CTLR-EL1--Interrupt-Controller-Control-Register--EL1-

EOImode bit is the bit 1 of the ICC_CTLR_EL1, Interrupt Controller Control Register (EL1)

Reply:
Section 12.2.6 ICC_CTLR_EL1, Interrupt Controller Control Register (EL1) of GICv3 spec

Reply:
@yyshen, It's bit 9 in the MMIO interface (GICC_CTLR), but bit 1 in the system register interface (ICC_CTLR).